### PR TITLE
add validation for start and reveal auction

### PR DESCRIPTION
**Pull request by GoodVincentTu**

### CODE REVIEW

File: src/lib/ensService.js [hunk 1]
```diff
@@ -124,6 +124,20 @@ export const sealedBids = (name, ether, secret, privateKey) => {
...
+  }
+}
+
+export const validateRevealAuctionBid = (name, ether, secret, privateKey) => {
+  const result = sealedBids(name, ether, secret, privateKey);
+  if (result === '0x0000000000000000000000000000000000000000') {
+    return 'Invalid sealed bids.';
+  }
+}
+
```
Comment (chochinlu) [inline src/lib/ensService.js]: (Suggestion)   if result === xxx, return xxxx ,   so the default return is ??

Comment (GoodVincentTu) [reply src/lib/ensService.js]: the value would be undefined, so in the StartAuctionForm and RevealAuctionForm, i check if the value is undefined, is this ok?

File: src/lib/ensService.js [hunk 2]
```diff
@@ -124,6 +124,49 @@ export const sealedBids = (name, ether, secret, privateKey) => {
...
+    validate: true,
+    err: ''
+  }
+  if (!checkZeroSealedBids(name, ether, secret, privateKey)) {
+    returnObj.validate = false;
+    returnObj.err = 'Invalid auction bids.';
+  }
+  return returnObj;
+}
+
```
Comment (chochinlu) [inline src/lib/ensService.js]: (Suggestion) 

``` js
export const validateStartAuctionBid = (name, ether, secret, privateKey) => 
  checkZeroSealedBids(name, ether, secret, privateKey) 
    ? {validate: true}
    : {validate: false, err: 'Invalid auction bids.'};
```

File: src/lib/ensService.js [hunk 3]
```diff
@@ -124,6 +124,49 @@ export const sealedBids = (name, ether, secret, privateKey) => {
   return contracts.ethRegistrar.sealedBids(fromAddress, bid);
 }
 
+const checkZeroSealedBids = (name, ether, secret, privateKey) => {
+  return (sealedBids(name, ether, secret, privateKey) === '0x0000000000000000000000000000000000000000');
+}
```
Comment (chochinlu) [inline src/lib/ensService.js]: (Suggestion) 

1.  function name -->  `isZeroSealBid`
2. 
``` js 
const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
const checkZeroSealedBids = (name, ether, secret, privateKey) => 
  sealedBids(name, ether, secret, privateKey) === ZERO_ADDRESS );
```

Comment (GoodVincentTu) [reply src/lib/ensService.js]: done

File: src/lib/ensService.js (hunk 2 above)
Comment (GoodVincentTu) [reply src/lib/ensService.js]: done

**Pull request merged**